app/models/archived/intake/gyr_intake_2021.rb

```diff
@@ -338,6 +338,8 @@ def self.sti_name
   enum widowed: { unfilled: 0, yes: 1, no: 2 }, _prefix: :widowed
   enum wants_to_itemize: { unfilled: 0, yes: 1, no: 2, unsure: 3 }, _prefix: :wants_to_itemize
 
+  scope :completed_intakes, -> { joins(:tax_returns).where(tax_returns: {current_state: TaxReturnStateMachine::INCLUDED_IN_PREVIOUS_YEAR_COMPLETED_INTAKES})}
```

Comment:
would like a second opinion on whether it's ok to add a new scope to this archived model. in my mind they shouldn't really change ever but we did this same thing for the current model so this was a continuation of that pattern.